configure.ac

```diff
@@ -2,7 +2,7 @@
 # Process this file with autoconf to produce a configure script.
 
 AC_PREREQ([2.69])
-AC_INIT([trexio],[2.4.0],[https://github.com/TREX-CoE/trexio/issues])
+AC_INIT([trexio],[2.4.2],[https://github.com/TREX-CoE/trexio/issues])
```

Comment:
You need to also update version numbers in `CMakeLists.txt` and in `_version.py` of the Python API

Reply:
OK. I added the file version_memo.txt to remember all the changes to be done when updating the version number.